docs/install_guides/pypi-openvino-rt.md

```diff
@@ -25,7 +25,9 @@ The Intel® Distribution of OpenVINO™ toolkit for Linux\*:
 * Supported Operating Systems:
   - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
   - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
```

Comment:
Are we going to support 3.8 which is default version for 20.04?

Reply:
Yes, but not in 2021.3, because it requires CI changes, the current configuration doesn't include python38